cypress/integration/v2/features/dynamic-content/current-post.spec.js

```diff
@@ -0,0 +1,326 @@
+
+/**
+ * External dependencies
+ */
+import { registerTests } from '~stackable-e2e/helpers'
+import { containsRegExp } from '~common/util'
+
+describe( 'Dynamic Content Current Post', registerTests( [
+	matchPostDataValues,
+	adjustFieldOptions,
+	adjustFieldValues,
+	assertEmptyValues,
+] ) )
+
+const fields = {
```

Comment:
Let's not do this. I just saw the code and how this variable was used for this test. 
Here are some comments

- I would suggest to populate the corresponding field values first before doing any steps for adding dynamic content and asserting values.
- We need to future-proof this code in case more fields were added. This approach already assumes that values can be accessed in `getCurentPost`. Use a different data structure for `fields`.